src/schemas/json/rust-project.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "crate": {
+      "type": "object",
```

Comment:
Optional fix (available in multiple places): add `title` property to make VS Code editing experience better:
Note: Red Hat `YAML` extension relies on `title` property to show it name when smth incorrect.